packages/utilities/psammead-locales/moment/ar.js

```diff
@@ -1,12 +1,11 @@
-/* eslint-disable func-names */
-const moment = require('moment');
+var moment = require('moment');
```

Comment:
As discussed on Slack, is there a way we can selectively allow `const` as IE11 doesn't seem to mind this?

Reply:
i don't think there  is way we can achieve this whilst still using ES5 as this will bring about IIFE block scoping isssues